explorer/index.html

```diff
@@ -0,0 +1,133 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset='utf-8' />
+  <title>Explore OSM without borders</title>
+  <meta name='viewport' content='initial-scale=1,maximum-scale=1,user-scalable=no' />
+  <script src='https://api.tiles.mapbox.com/mapbox-gl-js/v0.43.0/mapbox-gl.js'></script>
+  <link href='https://api.tiles.mapbox.com/mapbox-gl-js/v0.43.0/mapbox-gl.css' rel='stylesheet' />
+  <style>
+    body {
+      margin: 0;
+      padding: 0;
+    }
+    #map {
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      width: 100%;
+    }
+    #menu {
+      background: #fff;
+      position: absolute;
+      z-index: 1;
+      top: 10px;
+      right: 10px;
+      border-radius: 3px;
+      width: 120px;
+      border: 1px solid rgba(0, 0, 0, 0.4);
+      font-family: 'Open Sans', sans-serif;
+    }
+    #menu a {
+      font-size: 13px;
+      color: #404040;
+      display: block;
+      margin: 0;
+      padding: 0;
+      padding: 10px;
+      text-decoration: none;
+      border-bottom: 1px solid rgba(0, 0, 0, 0.25);
+      text-align: center;
+    }
+    #menu a:last-child {
+      border: none;
+    }
+    #menu a:hover {
+      background-color: #f8f8f8;
+      color: #404040;
+    }
+    #menu a.active {
+      background-color: #3887be;
+      color: #ffffff;
+    }
+    #menu a.active:hover {
+      background: #3074a4;
+    }
+  </style>
+</head>
+
+<body>
+  <nav id="menu"></nav>
+  <div id='map'></div>
+
+  <script>
+    var map = new mapboxgl.Map({
+      container: 'map',
+      style: 'https://openmaptiles.github.io/positron-gl-style/style-cdn.json',
+      hash: true,
+      center: [6.033, 49.678],
+      zoom: 5
+    });
+    var popup = new mapboxgl.Popup({
+      closeButton: false
+    });
+    map.on("load", function() {
+      map.addLayer({
+        'id': "all",
+        'type': 'fill',
+        'source-layer': 'vector-zones',
+        'source': {
+          'type': 'vector',
+          "tiles": ["http://localhost:8585/tiles/cosmogony/{z}/{x}/{y}.pbf"]
+        },
+        'layout': {
+          'visibility': 'visible'
+        },
+        "filter": ["==", "admin_level", 2],
+        'paint': {
+          'fill-color': '#088',
+          'fill-outline-color': "blue",
+          'fill-opacity': 0.8
+        }
+      });
+      map.on('mousemove', "all", function(e) {
+        map.getCanvas().style.cursor = 'pointer';
+        var feature = e.features[0];
+        popup.setLngLat(e.lngLat)
+          .setText(feature.properties.name)
+          .addTo(map);
+      });
+      map.on('mouseleave', "all", function() {
+        map.getCanvas().style.cursor = '';
+        popup.remove();
+      });
+      var toggleableLayerIds = [{
+        'level': 10,
+        'label': 'level10'
+      }, {
+        'level': 9,
+        'label': 'level9'
+      }, {
+        'level': 8,
+        'label': 'level8'
+      }, {
+        'level': 6,
+        'label': 'level6'
+      }, {
```

Comment:
we should also add the `level4`

Reply:
Yes, or using `zone_type`.
But that's maybe out of scope here, @Julien-laville is working on a more advanced UI on another branch.